### PR TITLE
update dashes, fix workload dash metric bug

### PR DESCRIPTION
**Pull request by RobDolares**
Signed-off-by: rhowingt@redhat.com <rhowingt@redhat.com>

### CODE REVIEW

File: manifests/base/grafana/dash-k8s-apiserver.yaml [hunk 1]
```diff
@@ -977,14 +981,16 @@ data:
...
+        "kubernetes-mixin"
+      ],
       "templating": {
         "list": [
           {
             "current": {
               "selected": false,
-              "text": "Observatorium",
-              "value": "Observatorium"
+              "text": "Prometheus",
```
Comment (marcolan018) [inline manifests/base/grafana/dash-k8s-apiserver.yaml:992]: why change from Observatorium to Prometheus?

File: manifests/base/grafana/dash-k8s-compute-resources-pod.yaml [hunk 2]
```diff
@@ -1914,8 +1916,8 @@ data:
             "allValue": null,
             "current": {
               "selected": false,
-              "text": "cert-manager-webhook-559cdf74cf-2hml5",
-              "value": "cert-manager-webhook-559cdf74cf-2hml5"
+              "text": "acm-custom-registry-d4f6cfc6d-f4trs",
```
Comment (marcolan018) [inline manifests/base/grafana/dash-k8s-compute-resources-pod.yaml:1919]: should we keep text/value as empty string in dashboard yaml?

File: manifests/base/grafana/dash-k8s-apiserver.yaml (hunk 1 above)
Comment (RobDolares) [reply manifests/base/grafana/dash-k8s-apiserver.yaml:992]: Probably changed by default from development instance.

File: manifests/base/grafana/dash-k8s-compute-resources-pod.yaml (hunk 2 above)
Comment (RobDolares) [reply manifests/base/grafana/dash-k8s-compute-resources-pod.yaml:1919]: I think so yes - likely exists in all of the current dashboards - will update.

**Pull request merged**